app/routers/symbol_settings.py

```
@@ -0,0 +1,823 @@
+    if not deleted:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Settings not found for symbol: {symbol}",
+        )
+
+
+@router.get("/symbols/{symbol}/estimated-cost", response_model=EstimatedCostResponse)
+async def get_estimated_cost(
+    symbol: str,
+    request: Request,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)